frontend/src/component/common/Table/cells/LinkCell/LinkCell.tsx

```diff
@@ -10,12 +10,14 @@ import {
     StyledTitle,
     StyledDescription,
 } from './LinkCell.styles';
+import { Tooltip } from '@mui/material';
 
 interface ILinkCellProps {
     title?: string;
     to?: string;
     onClick?: () => void;
     subtitle?: string;
+    withTooltip?: boolean;
```

Comment:
I'd avoid adding those special case boolean flags for configuring components and compose instead.